src/diffusers/__init__.py

```diff
@@ -32,7 +42,7 @@
     "loaders": ["FromOriginalModelMixin"],
     "models": [],
     "pipelines": [],
-    "quantizers.quantization_config": ["BitsAndBytesConfig", "GGUFQuantizationConfig", "TorchAoConfig"],
```

Comment:
Replacing direct imports here with dummy objects. It's a better guard for cases where imports in the quant configs might break the main diffusers import.

Reply:
Nice. Very specific and thorough!